ui/src/app/cohort-search/modifier-page/modifier-page.component.ts

```diff
@@ -309,4 +309,12 @@ export class ModifierPageComponent implements OnInit, OnDestroy, AfterContentChe
     const control = index === 0 ? 'valueA' : 'valueB';
     this.dateObjs[index] = new Date(this.form.get(['eventDate', control]).value + 'T08:00:00');
   }
+
+  numberValidation(event) {
+    if (!((event.keyCode > 95 && event.keyCode < 106)
+      || (event.keyCode > 47 && event.keyCode < 58)
+      || event.keyCode === 8)) {
+      return false;
+    }
+  }
```

Comment:
This prevents invalid input but I'm still able to paste in negative values. In that case, there should be an error message similar to the attributes page and the calculate button disabled.

Reply:
Also, we should probably be consistent with how we're doing validation across the app (we're handling it differently in attributes and review) but those need to be cleaned up anyway, especially attributes, so we can address that on a follow-up PR

Reply:
Made changes for modifier. No copy paste of negative number will show error message.